api/core/helpers.py

```diff
@@ -46,5 +40,22 @@ def send_login_email(user, request, *, welcome):
     return count
 
 
+def get_login_url(request, user):
+    base = reverse('redirector', args=["login"], request=request)
+    token = get_query_string(user)
+    return base + token
+
+
+def get_site_url(request):
+    return reverse('index', request=request)
+
+
+def get_unsubscribe_url(request, user):
+    # TODO: Change "login" to a new frontend path
+    base = reverse('redirector', args=["login"], request=request)
```

Comment:
We can change "login" to "unsubscribed" or whatever path gets implemented on the frontend.

Reply:
State added to frontend.